tests/test_constraint_manager.py

```diff
@@ -379,21 +591,48 @@ def _create_multi_model_both_constraints(self):
                 constraints:
                   perf_throughput:
                     min: 100
-              model_C:
+        """)
+        config = evaluate_mock_config(args, yaml_str, subcommand="profile")
+
+        return config
+
+    def _create_multi_model_with_global_constraints(self):
```

Comment:
change to with_matching_global_constraints

Reply:
Done 👍